src/test/java/org/folio/marc4ld/authority/field100/MarcToBibframeAuthorityConcept100IT.java

```diff
@@ -48,11 +47,8 @@ void shouldMapField100(ResourceTypeDictionary resourceType, String file) {
       .singleElement()
       .satisfies(resource -> validateRootResource(resource, List.of(CONCEPT, resourceType)))
       .satisfies(resource -> validateFocus(resource, resourceType))
-      .satisfies(this::validateForm)
-      .satisfies(this::validateTopic)
-      .satisfies(this::validateTemporal)
-      .satisfies(this::validatePlace)
-      .satisfies(this::validateIdentifier);
+      .satisfies(resource -> validateSubfocusResources(resource, FIELD_CODES))
+      .satisfies(resource -> validateIdentifierAndLccn(resource, "010fieldvalue"));
```

Comment:
Rename to `validateIdentifier`  (or `validateLccn`)

Reply:
fixed.